From 5d3428de974d15357b0ad407f4c5222cfaa8f9f3 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Mon, 8 Nov 2010 15:24:55 +1000
Subject: [PATCH] Add XF86XK_TouchpadOn/Off

Those keysyms will be used to report events from the hardware. Hardware
like the HP laptops emit 2 separate keycodes when the touchpad is enabled
or disabled. So we can catch those in user-space and display a popup.

https://bugs.freedesktop.org/show_bug.cgi?id=31300

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 XF86keysym.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/XF86keysym.h b/XF86keysym.h
index 200f5c1..fd3af4f 100644
--- a/xproto-7.0.19/XF86keysym.h
+++ b/xproto-7.0.19/XF86keysym.h
@@ -207,6 +207,8 @@
 #define XF86XK_Suspend		0x1008FFA7   /* Sleep to RAM                */
 #define XF86XK_Hibernate	0x1008FFA8   /* Sleep to disk               */
 #define XF86XK_TouchpadToggle	0x1008FFA9   /* Toggle between touchpad/trackstick */
+#define XF86XK_TouchpadOn	0x1008FFB0   /* The touchpad got switched on */
+#define XF86XK_TouchpadOff	0x1008FFB1   /* The touchpad got switched off */
 
 /* Keys for special action keys (hot keys) */
 /* Virtual terminals on some operating systems */
-- 
1.7.3.2

